distutils/sysconfig.py

```diff
@@ -280,25 +281,11 @@ def get_config_h_filename():
     return os.path.join(inc_dir, 'pyconfig.h')
 
 
-# Allow this value to be patched by pkgsrc. Ref pypa/distutils#16.
-_makefile_tmpl = 'config-{python_ver}{build_flags}{multiarch}'
-
-
 def get_makefile_filename():
     """Return full pathname of installed Makefile from the Python build."""
     if python_build:
```

Comment:
I also notice that the sysconfig version has a _PYTHON_BUILD with similar logic. Can the entire function be delegated to sysconfig?

Reply:
I wanted to keep this low-impact. But yes, I agree that Python probably knows better where its files are when run uninstalled. I can make that change if wanted.

Reply:
What if I add a CI job running the test suite with an uninstalled CPython? (this would require building CPython, ensurepip, install pytest, run tests)

Reply:
I don’t feel strongly about my suggestion. Happy to take baby steps too.

Regarding CI, I’d be on board if slightly reluctant due to the slowness and maintenance burden. At some point, all tests should be migrated to Setuptools, so I dread slightly added complication for that migration.

That said, better to have something useful to migrate than no protection at all. If you’re willing to contribute it, I’d accept it.

Reply:
OK, thanks. I went with delegating everything to sysconfig now.

I tried running the tests uninstalled, but there is one (unrelated) error I don't understand yet, so I'll keep this in mind but wont pursue for now.